shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/env/IntegrationTestEnvironment.java

```diff
@@ -37,7 +44,7 @@ public final class IntegrationTestEnvironment {
     
     private final Properties props;
     
-    private final ITEnvTypeEnum itEnvType;
+    private final ScalingITTypeEnum itType;
```

Comment:
`itType` could be recoverd to `itEnvType`, keep it the same as `scaling.it.env.type`